src/test/java/org/xstefank/model/PersistentListTest.java

```diff
@@ -16,7 +15,7 @@
     private static File userListFile = new File(TestUtils.TARGET_DIR, FILE_NAME);
 
     private final String USERNAME = "testUser";
-    private PersistentList testUserList;
+    private List<String> testUserList;
```

Comment:
here you can keep persistenList as this unit test is testing precisely PersistentList